part1/medplanner.py

```diff
@@ -0,0 +1,40 @@
+
+
+output = []
+# Function to generate welcome message
+def welcome():
+    username = input("What is your name? ")
+    output.append("Welcome " + str(username) + ", here is a look at this week")
+
+# Function to print Output
+
+def printOutput(x):
+    for item in x:
+        print(item)
+
+# write a function to generate the header row of the table and add it to output list
+
+def headerRow():
```

Comment:
Use F string in this function as well because your output is not currently aligned properly